### PR TITLE
 Add Example for Slider Labels in Taipy GUI

### PR DESCRIPTION
**Pull request by JeevaRamanathan**
Added an example demonstrating how to implement slider labels using a dictionary and lov in Taipy GUI.

### CODE REVIEW

File: doc/gui/examples/controls/slider_labels.py
Comment (FredLL-Avaiga) [inline doc/gui/examples/controls/slider_labels.py:1]: I think the usual motto about example is: keep it simple
ie demonstrate 1 thing in 1 example
I see 2 things here :-)
I would create 2 examples

**Review by FredLL-Avaiga — CHANGES_REQUESTED** [review]

File: doc/gui/examples/controls/slider_labels.py
Comment (JeevaRamanathan) [reply doc/gui/examples/controls/slider_labels.py:1]: Yeah sure, let me split those examples in two files.

Comment (JeevaRamanathan) [reply doc/gui/examples/controls/slider_labels.py:1]: I have changed them

File: doc/gui/examples/controls/slider_labels_dictionary.py
Comment (FabienLelaquais) [inline doc/gui/examples/controls/slider_labels_dictionary.py]: I don't think you need min and max: values are based on the dict keys.

File: doc/gui/examples/controls/slider_labels_dictionary.py
Comment (FabienLelaquais) [inline doc/gui/examples/controls/slider_labels_dictionary.py]:
```suggestion
labels = {
    0: "One",
    1: "Two",
    2: "Three",
    3: "Four",
    4: "Five",
    5: "Six",
```

really showing the added value of key/value pairs.

File: doc/gui/examples/controls/slider_labels_dictionary.py
Comment (FabienLelaquais) [inline doc/gui/examples/controls/slider_labels_dictionary.py]:
```suggestion
```
This line is useless: that would be the tab's title.

File: doc/gui/examples/controls/slider_labels_dictionary.py
Comment (FabienLelaquais) [inline doc/gui/examples/controls/slider_labels_dictionary.py]:
```suggestion
<|{value}|slider|labels={labels}|>
```

File: doc/gui/examples/controls/slider_labels_lov.py
Comment (FabienLelaquais) [inline doc/gui/examples/controls/slider_labels_lov.py]: Useless comment: that would be in the doc

File: doc/gui/examples/controls/slider_labels_lov.py
Comment (FabienLelaquais) [inline doc/gui/examples/controls/slider_labels_lov.py]:
```suggestion
<|{value}|slider|lov=One;Two;Three;Four;Five|labels|>
```
No need for '=True' in Markdown.
More meaningful labels.

File: doc/gui/examples/controls/slider_labels_lov.py
Comment (FabienLelaquais) [inline doc/gui/examples/controls/slider_labels_lov.py]: Useless line

**Review by FabienLelaquais — DISMISSED** [review]
Even slightly simpler...

Really helpful, @JeevaRamanathan, thanks!

File: doc/gui/examples/controls/slider_labels_dictionary.py
Comment (FabienLelaquais) [reply doc/gui/examples/controls/slider_labels_dictionary.py]: => `<|{value}|slider|labels={labels}|>`

Comment (JeevaRamanathan) [reply doc/gui/examples/controls/slider_labels_dictionary.py]: image
The slider default max value is defined `100`, so without them it results like this.

File: doc/gui/examples/controls/slider_labels_lov.py
Comment (JeevaRamanathan) [reply doc/gui/examples/controls/slider_labels_lov.py]: Except the first one, fixed remaining according to the suggestions

**Comment by JeevaRamanathan** [comment]
Hi @FabienLelaquais, I have fixed according to your suggestions expect https://github.com/Avaiga/taipy/pull/2077#pullrequestreview-2375714887

File: doc/gui/examples/controls/slider_labels_dictionary.py
Comment (FredLL-Avaiga) [inline doc/gui/examples/controls/slider_labels_dictionary.py]: as @FabienLelaquais mentioned, can you remove min and max ?
and change the labels dictionary to spread from 0 to 100 ?

**Review by FredLL-Avaiga — CHANGES_REQUESTED** [review]

File: doc/gui/examples/controls/slider_labels_dictionary.py
Comment (jeem5110) [reply doc/gui/examples/controls/slider_labels_dictionary.py:32]: Sure!

Comment (JeevaRamanathan) [reply doc/gui/examples/controls/slider_labels_dictionary.py]: Sure @FredLL-Avaiga

**Comment by JeevaRamanathan** [comment]
Hi @FredLL-Avaiga, the requested changes have been made. Please take a look, and let me know if any further adjustments are needed based on your time.

**Review by FredLL-Avaiga — APPROVED** [review]
Well done @JeevaRamanathan

**Comment by JeevaRamanathan** [comment]
Hi @FredLL-Avaiga, 
Are we good to merge this PR before Hacktoberfest, maybe after getting approval from @FabienLelaquais?

**Comment by FabienLelaquais** [comment]
Yes indeed we can accept this.
I'm not sure we need the second example though: I see no added value compared to the 'Select among a list of values' example already in the doc. But we will certainly reuse the first one.
Well done @JeevaRamanathan! and thank you.

File: doc/gui/examples/controls/slider_labels_lov.py
Comment (FredLL-Avaiga) [inline doc/gui/examples/controls/slider_labels_lov.py:1]: Can you remove this example as per @FabienLelaquais 's request ?

**Review by FredLL-Avaiga — CHANGES_REQUESTED** [review]

File: doc/gui/examples/controls/slider_labels_lov.py
Comment (JeevaRamanathan) [reply doc/gui/examples/controls/slider_labels_lov.py:1]: Hi @FredLL-Avaiga, I have removed this example, Thanks!

**Review by FredLL-Avaiga — APPROVED** [review]

**Comment by FredLL-Avaiga** [comment]
@jrobinAV can you merge this ?

**Comment by jrobinAV** [comment]
@JeevaRamanathan Thank you for your contribution !!

**Comment by RymMichaut** [comment]
@JeevaRamanathan, congratulations on this merged PR.
Could you please contact me by mail to prepare your swag? rym.michaut@taipy.io

**Comment by JeevaRamanathan** [comment]
I appreciate your help & support @FredLL-Avaiga,  @FabienLelaquais, @joaoandre, @RymMichaut! Thank you very much!

**Pull request merged**
